packages/manager/modules/vps/src/dashboard/vps-dashboard.html

```diff
@@ -11,10 +11,7 @@
             >
             </vps-dashboard-tile-information>
         </div>
-        <div
-            class="col-xm-12 col-md-4"
-            data-ng-if="!$ctrl.vps.isExpired && !$ctrl.isVpsNewRange"
-        >
+        <div class="col-xm-12 col-md-4" data-ng-if="!$ctrl.vps.isExpired">
```

Comment:
This change is made to ease testing purpose, it will be removed once all upgrades features are validated